XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -139,17 +139,26 @@ private void EnabledRow(Host host, UpdateType type, int index)
             {
                 selectedHosts = SelectedUpdateAlert.DistinctHosts;
             }
-            else if(FileFromDiskAlert != null) 
+            else if (FileFromDiskAlert != null)
             {
                 selectedHosts = FileFromDiskAlert.DistinctHosts;
             }
-            if(type != UpdateType.NewSuppPack && !host.CanApplyHotfixes)
+
+            Pool pool = Helpers.GetPoolOfOne(host.Connection);
+            if (pool != null && pool.IsPatchingForbidden)

```

Comment:
Is this restriction applicable to the installation of supplemental packs as well?